src/context/Firebase/Auth/context.tsx

```diff
@@ -10,11 +10,15 @@ export interface FirebaseContextProps {
   externalAuthSignIn?: (auth: Auth, provider: AuthProvider) => void
   logOut?: () => void
   currentUser: User | null
+  isAdmin: boolean
+  getUserToken: () => Promise<void>
```

Comment:
Consider refactoring this to expose the state rather than a getter, can also expose a `revalidateUser` function or something?

Reply:
`getUserToken` refreshes the user token which basically is the `revalidateUser` function and retrieves it from the server so im confused why we need to store the token in a state when the `admin` in the user token is already being stored in a state. We can always modify the `isAdmin` state to store the token. 
I guess the function name is a little bit confusing. It should be named `refreshUserToken` instead.